its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/SolutionKindTest.java

```diff
@@ -147,15 +152,17 @@ void testCSharpSharedProjectType() throws IOException {
   void testCSharpFramework48() throws IOException {
     var folderName = "CSharp.Framework.4.8";
     assumeFalse(TestUtils.getMsBuildPath(ORCHESTRATOR).toString().contains("2017")); // We can't run .NET Core SDK under VS 2017 CI context
-    BuildResult buildResult = runAnalysis(folderName, true);
+    var analysisContext = AnalysisContext.forServer(folderName);
+    TestUtils.runNuGet(analysisContext.orchestrator, analysisContext.projectDir, false, "restore");
```

Comment:
There is a TODO on `TestUtils.runNuGet` but I didn't touched it as part of the PR. Let me know if I should do. I need a bit of instructions about how you think this should be done.

Reply:
Is this this project that fails to restore with msbuild, or can we simply remove the nuget from here?

I'm not sure if this was a troublemaker in the past, or if it's simply a copy/paste of scaffolding that is more complicated than it needs to be.

Reply:
FYI: #2415 that is approved changes `BuildCommand` implementation. While it shouldn't really break, if it does, it might be due to that.

Reply:
> Is this this project that fails to restore with msbuild, or can we simply remove the nuget from here?

Yes. It fails and needs the nuget restore.